examples/asr/conf/finetune/fintune_with_existing_tokenizer.yaml

```diff
@@ -0,0 +1,101 @@
+name: "FastConformer-Transducer-BPE"
+
+init_from_pretrained_model: ?? # pretrained model name or path to the pretrained model checkpoint
+
+model:
```

Comment:
Add spec augment section, default values as none, and defect if none in code to copy the values from the models config.

Reply:
I think better option is to default the current values we use in finetuning yaml file and if its none then skip. Otherwise users won;t have option to skip spec_aug.